### DISCUSSION TITLE
Component对应deployment已经启动成功了，但是application状态还是显示running，Phase:wait

### DISCUSSION THREAD

**Discussion by wangqi811** [kubevela/kubevela#5972] [Q&A]
![image](https://github.com/kubevela/kubevela/assets/15891191/70d8dbe9-9c8d-4a38-8b2e-d90d1223f8cd)

**Comment by wangqi811** [comment]
vela-core版本：1.2.5

**Comment by wangqi811** [comment]
@Somefive 麻烦问一下，我设置的isHealth条件为：  
status:
    healthPolicy: |
      isHealth: context.output.status.readyReplicas == context.output.status.replicas

但是现在观察到output的deployment满足这个状态之后，查看application status会发现对应component的heathy:false，这个大概是什么原因导致的呢？

**Reply by Somefive** [reply]
可能会有延迟？因为这个条件满足并不会立刻触发应用感知到这个事情，应用会隔一段时间检查一下。如果一直没有变化那就得到 core 里面排查一下这个配置的问题了，会有一点复杂。

**Reply by wangqi811** [reply]
现在观察下来是会有延迟，延迟时间不确定，也可能一直不更新，我现在新增了一个定时任务，定期轮训runningWorkflow状态的application，检测到就稍微改动某个组件的参数触发新的调谐，观察下来是可以达到目的的。
